lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
             try
             {
                 TangFactory.GetTang()
-                    .NewInjector(mapInputPipelineDataConverterConfig)
+                    .NewInjector(mapOutputPipelineDataConverterConfig)
                     .GetInstance<IPipelineDataConverter<TMapOutput>>();
```

Comment:
Would it matter if a wrong configuration is given? When you put wrong one, it didn't impact any test case. Is it because test case doesn't cover this scenario or because a wrong configuration also works?